langgraph_supervisor/supervisor.py

```diff
@@ -81,12 +81,12 @@ def _process_output(output: dict) -> dict:
             "messages": messages,
         }
 
-    def call_agent(state: dict) -> dict:
-        output = agent.invoke(state)
+    def call_agent(state: dict, config: Optional[dict] = None) -> dict:
```

Comment:
let's use `RunnableConfig` for the typing (and i don't think we need optional/None